pkg/filesystem/subvolume.go

```diff
@@ -87,7 +158,14 @@ func listCephFSSubvolumes(ctx context.Context, clientsets *k8sutil.Clientsets, o
 			continue
 		}
 		for _, svg := range subvolg {
-			svList, err := exec.RunCommandInOperatorPod(ctx, clientsets, "ceph", []string{"fs", "subvolume", "ls", fs.Name, svg.Name}, operatorNamespace, clusterNamespace, true)
+			cmd := "ceph"
+			args := []string{"fs", "subvolume", "ls", fs.Name, svg.Name, "--format", "json"}
+
+			if checkForExternalStorage(ctx, clientsets, clusterNamespace) {
+				m, admin_id, admin_key := getExternalClusterDetails(ctx, clientsets, operatorNamespace, clusterNamespace)
+				args = []string{"fs", "subvolume", "ls", fs.Name, svg.Name, "--format", "json", "-m", m, "--id", admin_id, "--key", admin_key}
```

Comment:
Do we need to run this CLI from the machine where the ceph packages are installed?